credsweeper/validations/github_token_validation.py

```diff
@@ -38,7 +38,7 @@ def verify(cls, line_data_list: List[LineData]) -> KeyValidationOption:
                 headers={"Authorization": f"token {line_data_list[0].value}"},
             )
         except (requests.exceptions.ConnectionError, Exception) as exc:
-            logger.info(exc)
+            logger.error(exc)
```

Comment:
Please describe why do we weed that change, and how does it related to gzip archives mentioned in the PR description

Reply:
It was found during debug fuzzing  #188